Documentation/Installation.MD

```diff
@@ -10,13 +10,13 @@ Add this entry in your `Podfile`:
 
 ```ruby
 # Installs both StreamChat and StreamChatUI frameworks
-pod 'StreamChatUI', `~> 3.1`
+pod 'StreamChatUI', '~> 3.1'
 ```
 
 ```ruby
 # In case you don't need the UI framework (you're building the UI layer yourself)
 # you can specify the StreamChat framework directly.
-pod 'StreamChat', `~> 3.1`
+pod 'StreamChat', '~> 3.1
```

Comment:
I think the ending quote is missing here

Reply:
I should've named this PR "break quote marks in a different way"